plugins/gateway/kong/plugin.go

```diff
@@ -65,13 +73,21 @@ func createTelemetry(kong *pdk.PDK) (*models.Telemetry, error) {
 	truncatedBodyReq := false
 	truncatedBodyRes := false
 
+	requestTime, err := getRequestTimeFromContext(kong)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get request time from context %v", err)
+	}
+	responseTime, err := common.GetTimeNowRFC3339Nano()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get time now: %v", err)
+	}
 	routedService, err := kong.Router.GetService()
 	if err != nil {
 		return nil, fmt.Errorf("failed to get routed serivce: %v", err)
 	}
-	clientIP, err := kong.Client.GetIp()
+	clientIP, err := kong.Client.GetForwardedIp()
 	if err != nil {
-		return nil, fmt.Errorf("failed to get client ip: %v", err)
+		return nil, fmt.Errorf("failed to get client forwarded ip: %v", err)
```

Comment:
do we want to fail the telemetry on that error? why this error can happen?

Reply:
IMO we should not fail the telemetry on that error. No idea what can cause this